plugins/flatpak/gs-flatpak.c

```diff
@@ -214,6 +215,46 @@ remote_is_eos_apps (FlatpakRemote *remote)
 }
 
 static gboolean
+gs_flatpak_is_removable_remote (GsFlatpak *self,
```

Comment:
I believe that you could also implement this by using `g_file_find_enclosing_mount()` on the remote URL and avoid enumerating all the drives from the volume monitor.